client/components/homepage/contact/ContactItem.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-const ContactItem = (props) => (  
+const ContactItem = props => (
   <div className="col-xs-12 col-sm-6 col-md-3 text-center">
     <div className="box-simple">
       <img src={`../../img/${props.name}.jpg`} className="icon" />
```

Comment:
I can go back and enlarge the images to make them more prominent. Didn't know that was on the agenda of things to do 😄